cmake can't find CoinUtils, Clp or Osi

Hi,

I'm  trying to compile the rc3.0 branch (commit 9a16c3), but cmake can't find CoinUtils, Clp or Osi:


```
~/AliceVision/build on rc3.0 $ cmake \ 
-DCMAKE_BUILD_TYPE=Release \ 
-D{COINUTILS,CLP,OSI}_INCLUDE_DIR_HINTS=/usr/include/coin \
-DCMAKE_INSTALL_PREFIX=/usr \ 
-DCMAKE_INSTALL_LIBDIR=lib \ 
-DFLANN_INCLUDE_DIR_HINTS=/usr/include/flann \ 
-DLEMON_INCLUDE_DIR_HINTS=/usr/include/lemon \ 
-DALICEVISION_USE_CUDA=OFF \ 
-DALICEVISION_USE_OPENGV=ON \ 
-DALICEVISION_USE_OPENCV=ON \ 
-DALICEVISION_BUILD_DOC=ON \ 
-G Ninja \ 
..
...
CMake Error at src/CMakeLists.txt:443 (find_package): 
  By not providing "FindCoinUtils.cmake" in CMAKE_MODULE_PATH this project 
  has asked CMake to find a package configuration file provided by 
  "CoinUtils", but CMake did not find one. 
 
  Could not find a package configuration file provided by "CoinUtils" with 
  any of the following names: 
 
    CoinUtilsConfig.cmake 
    coinutils-config.cmake 
 
  Add the installation prefix of "CoinUtils" to CMAKE_PREFIX_PATH or set 
  "CoinUtils_DIR" to a directory containing one of the above files.  If 
  "CoinUtils" provides a separate development package or SDK, be sure it has 
  been installed.
```
Similar error messages show up for Clp or Osi.

However, CoinUtils, Clp and Osi are installed:
```
$ ls /usr/include/coin  
CbcOrClpParam.cpp            ClpDynamicMatrix.hpp     ClpPEDualRowDantzig.hpp        ClpSimplex.hpp              CoinFileIO.hpp            CoinPackedVectorBase.hpp     CoinPresolveSingleton.hpp    CoinStructuredModel.hpp      OsiChooseVariable.hpp 
CbcOrClpParam.hpp            ClpEventHandler.hpp      ClpPEDualRowSteepest.hpp       ClpSimplexDual.hpp          CoinFinite.hpp            CoinParam.hpp                CoinPresolveSubst.hpp        CoinTime.hpp                 OsiClpSolverInterface.hpp 
ClpCholeskyBase.hpp          ClpFactorization.hpp     ClpPEPrimalColumnDantzig.hpp   ClpSimplexNonlinear.hpp     CoinFloatEqual.hpp        CoinPragma.hpp               CoinPresolveTighten.hpp      CoinTypes.hpp                OsiColCut.hpp 
ClpCholeskyDense.hpp         ClpGubDynamicMatrix.hpp  ClpPEPrimalColumnSteepest.hpp  ClpSimplexOther.hpp         CoinHelperFunctions.hpp   CoinPresolveDoubleton.hpp    CoinPresolveTripleton.hpp    CoinUtility.hpp              OsiCollections.hpp 
ClpCholeskyPardiso.hpp       ClpGubMatrix.hpp         ClpPESimplex.hpp               ClpSimplexPrimal.hpp        CoinIndexedVector.hpp     CoinPresolveDual.hpp         CoinPresolveUseless.hpp      CoinUtilsConfig.h            OsiConfig.h 
ClpConfig.h                  ClpInterior.hpp          ClpPackedMatrix.hpp            ClpSolve.hpp                CoinLpIO.hpp              CoinPresolveDupcol.hpp       CoinPresolveZeros.hpp        CoinWarmStart.hpp            OsiCut.hpp 
ClpConstraint.hpp            ClpLinearObjective.hpp   ClpParameters.hpp              Clp_C_Interface.h           CoinMessage.hpp           CoinPresolveEmpty.hpp        CoinRational.hpp             CoinWarmStartBasis.hpp       OsiCuts.hpp 
ClpConstraintLinear.hpp      ClpMatrixBase.hpp        ClpPdcoBase.hpp                CoinAlloc.hpp               CoinMessageHandler.hpp    CoinPresolveFixed.hpp        CoinSearchTree.hpp           CoinWarmStartDual.hpp        OsiPresolve.hpp 
ClpConstraintQuadratic.hpp   ClpMessage.hpp           ClpPlusMinusOneMatrix.hpp      CoinBuild.hpp               CoinModel.hpp             CoinPresolveForcing.hpp      CoinShallowPackedVector.hpp  CoinWarmStartPrimalDual.hpp  OsiRowCut.hpp 
ClpDualRowDantzig.hpp        ClpModel.hpp             ClpPresolve.hpp                CoinDenseFactorization.hpp  CoinModelUseful.hpp       CoinPresolveImpliedFree.hpp  CoinSignal.hpp               CoinWarmStartVector.hpp      OsiRowCutDebugger.hpp 
ClpDualRowPivot.hpp          ClpNetworkMatrix.hpp     ClpPrimalColumnDantzig.hpp     CoinDenseVector.hpp         CoinMpsIO.hpp             CoinPresolveIsolated.hpp     CoinSimpFactorization.hpp    Coin_C_defines.h             OsiSolverBranch.hpp 
ClpDualRowSteepest.hpp       ClpNode.hpp              ClpPrimalColumnPivot.hpp       CoinDistance.hpp            CoinOslFactorization.hpp  CoinPresolveMatrix.hpp       CoinSmartPtr.hpp             Idiot.hpp                    OsiSolverInterface.hpp 
ClpDummyMatrix.hpp           ClpNonLinearCost.hpp     ClpPrimalColumnSteepest.hpp    CoinError.hpp               CoinPackedMatrix.hpp      CoinPresolveMonitor.hpp      CoinSnapshot.hpp             OsiAuxInfo.hpp               OsiSolverParameters.hpp 
ClpDynamicExampleMatrix.hpp  ClpObjective.hpp         ClpQuadraticObjective.hpp      CoinFactorization.hpp       CoinPackedVector.hpp      CoinPresolvePsdebug.hpp      CoinSort.hpp                 OsiBranchingObject.hpp       OsiUnitTests.hpp 
```
 
The only way I got cmake to successfully finish, can be seen in my fork: https://github.com/triveria/AliceVision/tree/make-compilable-on-debian.
Here, the files src/cmake/FindClp.cmake, src/cmake/FindCoinUtils.cmake, src/cmake/FindOsi.cmake are readded from commit c78196db406b78d0ca4254e4839b7e377322a389.

I get exactly the same behaviour on Debian 12 (bookworm) and on Manjaro.

So I wonder, how do you people compile AliceVision? Do I need to add some more cmake flags so that CoinUtils and OR-tools can be found?

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -25,12 +25,15 @@ AliceVision depends on external libraries:
 * [Assimp >= 5.0.0](https://github.com/assimp/assimp)
 * [Boost >= 1.74.0](https://www.boost.org)
 * [Ceres >= 1.10.0](https://github.com/ceres-solver/ceres-solver)
+* CoinUtils >= 2.9.3 use [our fork](https://github.com/alicevision/CoinUtils) with a CMake build system so that it can be easily found
+* Coin-or linear programming (Clp) use [our fork](https://github.com/alicevision/Clp) with a CMake build system
 * [Eigen >= 3.3.4](https://gitlab.com/libeigen/eigen)
 * [Geogram >= 1.7.5](https://gforge.inria.fr/frs/?group_id=5833)
 * [Geogram >= 1.7.5](https://github.com/BrunoLevy/geogram)
 * [Expat >= 2.4.8](https://libexpat.github.io/)
 * [OpenEXR >= 2.5](https://github.com/AcademySoftwareFoundation/openexr)
 * [OpenImageIO >= 2.1.0 (recommended >= 2.4.6)](https://github.com/OpenImageIO/oiio)
+* Open Solver Interface (Osi) >= 0.106.10 use [our fork](https://github.com/alicevision/Osi)) with a CMake build system
 * [zlib](https://www.zlib.net)
 
 Other optional libraries can enable specific features (check "CMake Options" for enabling them):
@@ -46,9 +49,6 @@ Other optional libraries can enable specific features (check "CMake Options" for
 * PCL (Point Cloud Library) >= 1.12.1 for the registration module
 * PopSift (feature extraction on GPU)
 * UncertaintyTE (Uncertainty computation)
-* CoinUtils >= 2.9.3 (if you need to build it use [our fork](https://github.com/alicevision/CoinUtils) with a CMake build system)
-* Coin-or linear programming (Clp) (if you need to build it use [our fork](https://github.com/alicevision/Clp) with a CMake build system)
-* Open Solver Interface (Osi) >= 0.106.10 (if you need to build it use [our fork](https://github.com/alicevision/Osi))
 
 
 AliceVision also depends on some embedded libraries:

